.ci/.bump-go-release-version.yml

```diff
@@ -48,6 +50,7 @@ projects:
     overrideGoVersion: "1.17"
     enabled: true
     labels: dependency,backport-skip
+    description: It requires the version to be bumped first in the https://github.com/elastic/golang-crossbuild. Otherwise it will fail until the docker images are available.
```

Comment:
```suggestion
    description: It requires the version to be bumped first in golang-crossbuild project, then a new release will be added to https://github.com/elastic/golang-crossbuild/releases. Otherwise it will fail until the docker images are available.
```